pyros_schemas/ros/schemagic.py

```diff
@@ -51,6 +51,7 @@ def create(ros_msg_class,
 
     members_types = _get_rosmsg_fields_as_dict(ros_msg_class)
     members = {}
+    schema_instance = RosSchema()
```

Comment:
RosSchema() is not run with the members, so the __new__ and __init__ method are not doing what's expected. I didnt check the details but I m pretty sure it might cause problems...

Better to follow the documented API usage as much as possible : http://marshmallow.readthedocs.io/en/latest/api_reference.html#api-fields
There is inheritance and some metaclass here, it can be tricky to get right...

Reply:
You are right. There may be a problem.
But I only had to do this to fix the memory leak.
I simply needed a RosSchema instance
I just created it.